### PR TITLE
Addition of PACEA and 14C-Palaeolithic databases

### PR DESCRIPTION
**Pull request by yesdavid**
I added the PACEA and 14C-Palaeolithic databases, as announced in my [previous post](https://github.com/ropensci/c14bazAAR/issues/2#issuecomment-577190412).

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -9,6 +9,11 @@
 - added a basic write function for c14_date_lists: `write_c14()` (#84)
 - added a version column that documents from which database version a certain date is pulled (#85)
 
+
+## new getter functions
```
Comment (nevrome) [inline NEWS.md]: These are already changes in for 1.2.1 - you started the work on a new release

File: R/get_14cpalaeolithic.R [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  # db_url <-"https://ees.kuleuven.be/geography/projects/14c-palaeolithic/radiocarbon-palaeolithic-europe-database-v26-extract.xlsx"
+
+
+  # download data
+  temp <- tempfile(fileext=".xlsx")
+
+  utils::download.file(url = db_url, destfile = temp, mode="wb")
+
+  # Read each file and write it to csv
+  lapply(temp, function(f) {
```
Comment (nevrome) [inline R/get_14cpalaeolithic.R]: Why this extra step of reading xlsx, writing to csv and reading again from csv?

File: R/get_14cpalaeolithic.R [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+  db_path_xlsx <- unlist(strsplit(temp, split = ".", fixed = TRUE))
+  db_path_csv <- paste(db_path_xlsx[1], ".csv", sep = "")
+
+  # read data
+  db_raw <- data.table::fread(
+    db_path_csv,
+    sep = ",",
+    encoding = "UTF-8",
+    colClasses = "character",
```
Comment (nevrome) [inline R/get_14cpalaeolithic.R]: So this reads every column as character? That could probably do as well into other parser functions, or? Less verbose.

File: R/get_pacea.R [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    exdir = tempdir()
+  )
+
+  # read data
+  db_raw <- data.table::fread(
+    db_path,
+    sep = "\t",
+    encoding = "UTF-8",
+    skip = 1,
+    colClasses = c(
```
Comment (nevrome) [inline R/get_pacea.R]: You could use the solution from the other parser here as well, ja?

File: data-raw/url_reference.csv [hunk 5]
```diff
@@ -13,3 +13,5 @@ palmisano,2017-09-23,1,https://discovery.ucl.ac.uk/1575442/1/Palmisanoetal.zip
 radon,today,1,https://radon.ufg.uni-kiel.de/radondaily.txt
 radonb,today,1,https://radon-b.ufg.uni-kiel.de/radondaily.txt
 irdd,2018-08-13,1,https://zenodo.org/record/3367518/files/Chapple%2C%20RM%202019%20Catalogue%20of%20radiocarbon%20determinations%20%26%20dendrochronology%20dates%20%28August%202019%20Release%29.xlsx
+pacea,2020-01-22,1,http://www.paleoanthro.org/media/journal/content/PA20110001_S01.zip
+14c-palaeolithic2020-1-23,,1,https://ees.kuleuven.be/geography/projects/14c-palaeolithic/radiocarbon-palaeolithic-europe-database-v26-extract.xlsx
```
Comment (nevrome) [inline data-raw/url_reference.csv]: Please use the same database name: `14c-palaeolithic` -> `14cpalaeolithic`

There also seems to be a table layout error in `14c-palaeolithic2020-1-23,,1`

File: data-raw/url_reference.csv [hunk 6]
```diff
@@ -13,3 +13,5 @@ palmisano,2017-09-23,1,https://discovery.ucl.ac.uk/1575442/1/Palmisanoetal.zip
 radon,today,1,https://radon.ufg.uni-kiel.de/radondaily.txt
 radonb,today,1,https://radon-b.ufg.uni-kiel.de/radondaily.txt
 irdd,2018-08-13,1,https://zenodo.org/record/3367518/files/Chapple%2C%20RM%202019%20Catalogue%20of%20radiocarbon%20determinations%20%26%20dendrochronology%20dates%20%28August%202019%20Release%29.xlsx
+pacea,2020-01-22,1,http://www.paleoanthro.org/media/journal/content/PA20110001_S01.zip
```
Comment (nevrome) [inline data-raw/url_reference.csv]: Can we get an https url here? Everything else can cause nasty problems nowadays.

File: data-raw/variable_reference.csv [hunk 7]
```diff
@@ -1,48 +1,48 @@
...
-,,,,,,RECORD_SOURCE,,INCONGR,Contexto Funcional,,,,,Updater,Mode (cal BC/AD)
-,,,,,,DATE_ISSUES,,,Tipo de tumba,,,,,Date Updated,δ13C or 13C/12C ratio
-,,,,,,AGE_NORM,,,Ritual funerario,,,,,Measured Age,Measured radiocarbon date
-,,,,,,ADDITIONAL_DATA_ISSUES,,,,,,,,MA Sigma,1σ start
-,,,,,,,,,,,,,,Delta 13C Source,1σ finish
-,,,,,,,,,,,,,,Significance,2σ start
-,,,,,,,,,,,,,,Stratigraphic Component,2σ finish
-,,,,,,,,,,,,,,Context,"series order: 1, 3, 4, 2"
-,,,,,,,,,,,,,,Associated Taxa,Other References to look for
+c14bazAAR,type,definition,source,14sea,adrac,austarch,calpal,context,eubar,euroevol,palmisano,radon,radonb,kiteeastafrica,irdd,PACEA,14cpalaeolithic
```
Comment (nevrome) [inline data-raw/variable_reference.csv]: One standard database name everywhere please: `PACEA` -> `pacea`

File: R/get_14cpalaeolithic.R (hunk 2 above)
Comment (yesdavid) [reply R/get_14cpalaeolithic.R]: Because I was running into errors using the .xlsx-file

**Comment by yesdavid** [comment]
thank you @nevrome for your input!

**Pull request merged**